portal-frontend/src/app/features/notice-of-intents/notice-of-intent-details/roso-details/roso-details.component.scss

```diff
@@ -0,0 +1,18 @@
+@use '../../../../../styles/functions' as *;
+
+.soil-table {
+  display: grid;
+  overflow-x: auto;
+  grid-column-gap: rem(36);
+  grid-row-gap: rem(12);
+  grid-template-columns: max-content max-content;
+
+  @media screen and (min-width: $tabletBreakpoint) {
```

Comment:
Rather than repeat the same CSS / text over and over, have you thought about making a new component? We do have soil-table component for the proposal phase, so it might make sense to have one for the review/display phase as well.

Reply:
Yes, i and tristan were chatting about it. but i wasnt sure on why this was done in first place. so i rather focused on resolving it first and then adding a task to combine it all together after chatting with whoever developed this in first place.

Reply:
Each app type was built one at a time, so we did not know which app types would share functionality until many of them were implemented. Are you saying you have filed an improvement ticket to address this?

Reply:
I was planning to do it. bring it up in the coming backlog session after chatting with someone then.

Reply:
I'll approve the MR once the ticket is made.